Makefile

```diff
@@ -44,13 +44,25 @@ my.env:
 .docker-build:
 	make build
 
+.devcontainer-build:
+	make devcontainerbuild
+
 .PHONY: build
 build: my.env  ## | Build docker images.
 	${DC} build ${DOCKER_BUILD_OPTS} --build-arg userid=${SOCORRO_UID} --build-arg groupid=${SOCORRO_GID} --progress plain app
 	${DC} build --progress plain oidcprovider fakesentry
 	${DC} build --progress plain statsd postgresql memcached localstack elasticsearch symbolsserver
 	touch .docker-build
 
+.PHONY: devcontainerbuild
+devcontainerbuild: my.env  ## | Build VS Code development container.
+	${DC} build devcontainer
+	touch .devcontainer-build
+
+.PHONY: devcontainer
+devcontainer: .env .devcontainer-build  ## | Run VS Code development container.
```

Comment:
FWICT trying this locally, `.devcontainer-build` is empty (it's created in the above `devcontainerbuild` command) and doesn't appear to do anything -- is it doing something?

Reply:
to my knowledge absolutely nothing, I just copied the pattern from `.docker-build` assuming it was for make reasons.

@willkg can you elaborate on why `.docker-build` exists?

Reply:
`.docker-build` is a real file on the file system and has a rule for building it. As such, you can use it as a dependency in Make rules and if it doesn't exist, Make will run the `.docker-build` rule which will build the container.

Reply:
For example, you can start with a completely clean repo and do `make test` and Make will see there's no `.docker-build` file, run that rule building everything, and then run the tests.